tests/Benchmarks/System.Text.JsonLab/JsonWriterPerf.cs

```diff
@@ -37,15 +35,17 @@ public void Setup()
             var buffer = new byte[BufferSize];
             _stream = new MemoryStream(buffer);
             _writer = new StreamWriter(_stream, enc, BufferSize, true);
-            _symbolTable = GetTargetEncoder(Target);
-            _arrayFormatter = new ArrayFormatter(BufferSize, _symbolTable);
+            _arrayFormatter = new ArrayFormatter(BufferSize, GetTargetEncoder(Target));
         }
 
         [Benchmark]
         public void WriterSystemTextJsonBasic()
         {
             _arrayFormatter.Clear();
-            WriterSystemTextJsonBasic(Formatted, _arrayFormatter);
+            if (Target == EncoderTarget.InvariantUtf8)
```

Comment:
@adamsitnik, please review the changes in JsonWriterPerf.cs.

Is there a way for me to avoid adding this if check? It doesn't have much impact on the end result, though.

Reply:
The extra branch should not impact this particular benchmark (it's not a nano-benchmark).

To avoid this check you would have to remove `EncoderTarget` param and write a benchmark method per case